packages/pass-style/test/test-passStyleOf.js

```diff
@@ -41,7 +41,7 @@ test('some passStyleOf rejections', t => {
     message:
       /Only registered symbols or well-known symbols are passable: "\[Symbol\(unique\)\]"/,
   });
-  if (harden.isFake) {
+  if ('isFake' in harden) {
```

Comment:
What motivates these changes?

Reply:
the line started erroring because `harden` doesn't have an `isFake` property. I figure something in my changes fixed a resolution that had been falling back to `any`